brambling/templates/brambling/event/organizer/update.html

```diff
@@ -1,9 +1,295 @@
+{% load floppyforms %}
```

Comment:
I think structure-wise it can just be one file per view? Or maybe file for each group of related views. This is also the place where you can put tests for any other basic stuff (e.g. assert the response is 403 if the user is not logged in, or logged in as the wrong user) or actual application logic happening in the view code itself.